src/main/java/frc/robot/subsystems/climberSubsystem/ClimberSubsystem.java

```diff
@@ -50,16 +49,20 @@ private void configMotor(CANSparkMax motor, boolean invert) {
     pidController.setI(ClimberConstants.CURRENT_PID[1]);
     pidController.setD(ClimberConstants.CURRENT_PID[2]);
     double armPosition;
-    if(invert){
-      armPosition = 1-motor.getAbsoluteEncoder().getPosition();
-    }else{
-      armPosition = motor.getAbsoluteEncoder().getPosition();
-    }
 
-    motor.setSoftLimit(SoftLimitDirection.kForward,(float) (Units.degreesToRotations(135)*ClimberConstants.GEAR_RATIO));
+    armPosition = motor.getAbsoluteEncoder().getPosition();
+
+    motor.setSoftLimit(
+        SoftLimitDirection.kForward,
+        (float) (Units.degreesToRotations(135) * ClimberConstants.GEAR_RATIO));
     motor.setSoftLimit(SoftLimitDirection.kReverse, 0);
+    double motorPosition = armPosition * ClimberConstants.GEAR_RATIO;
+    if (Math.abs(motorPosition) > 10) {
+      motor.getEncoder().setPosition(0);
+    } else {
+      motor.getEncoder().setPosition(motorPosition);
+    }
```

Comment:
Lets make this account for the wraparound of the absolute sensor instead of this workaround

Reply:
Is not a blocker for this PR. A card is open for it, with Josh assigned to it.